src/core/bounds/bounds.utils.ts

```diff
@@ -67,7 +67,12 @@ export const calculateBounds = (
   const { centerZoomedOut } = contextInstance.setup;
 
   if (!wrapperComponent || !contentComponent) {
-    throw new Error("Components are not mounted");
+    return {
```

Comment:
Unfortunately, this causes `resetTransform` to not work

Reply:
How exactly? I mean this change should affect anything only at first render time to prevent this component throwing an error in case of a delayed render of a children.